contrib/docker/Dockerfile.oxend

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04
+
+ARG USER_ID
+ARG GROUP_ID
+
+COPY ./oxen-linux-x86_64-8.1.6/* /usr/local/bin/
```

Comment:
Using the binary distribution for this is going to be a bit wasteful: they are massive, full-static-libs binaries.  Since the container here is Ubuntu based you could pretty easily use an `apt-get -y install oxend oxen-wallet-cli` after setting up the apt repo in the container (see https://deb.oxen.io/) which would give you the latest version without having to update download versions in these scripts.  Those binaries are *considerably* smaller (and thus will end up using less memory, plus will get Ubuntu-provided security updates/fixes for dependent packages).

Reply:
Oh, duh. Yeah, I was going to use a smaller base image, then switched to Ubuntu to simplify things for the first pass. Using `apt-get` makes way more sense at this point.

Reply:
You could also try this docker image:

registry.oxen.rocks/lokinet-ci-ubuntu:focal

which is what we use for CI builds, and has all the dependencies pre-installed.  (It also has compilers and headers, so might end up being bigger, but it's worth a try).

Reply:
we should make a base image with the apt repos added to them and that's it

Reply:
Yeah, the CI image is quite a bit bigger, 1.12G vs. 154M.

Making a base image is a good idea.

Reply:
Split out `oxen-ubuntu` base image and build script. `oxend` container image now based on that, instead of `ubuntu` + statically-linked oxen binaries.